qtranslate_hooks.php

```diff
@@ -225,6 +225,9 @@ function qtranxf_wpseo_replacements($replacements){
 add_filter('link_description', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 add_filter('the_author', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 
+add_filter('get_terms', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
```

Comment:
I did test it. It probably works here because I imported the terms with my W2Q: WPML to qTranslate tool and I wasn't aware that terms were translated a different way. Any reason why terms are treated differently?